samples/tab-personal/mvc-csharp/AppManifest/manifest.json

```diff
@@ -32,7 +32,17 @@
       "contentUrl": "https://<<Base-url>>/PersonalTab",
       "websiteUrl": "https://<<Base-url>>/PersonalTab",
       "scopes": [
-        "personal"
+        "personal",
+        "groupChat"
+      ],
+      "context": [
+        "personalTab",
+        "channelTab",
```

Comment:
channelTab only works in the team scope. Since team is not listed in the scopes, then this context doesn't make sense.

Reply:
removed the property